scripts/run_backend_tests.py

```diff
@@ -81,7 +80,11 @@
 _LOAD_TESTS_DIR = os.path.join(os.getcwd(), 'core', 'tests', 'load_tests')
 
 
-_PARSER = argparse.ArgumentParser()
+_PARSER = argparse.ArgumentParser(description="""
+    Run this script from the oppia root folder:
+
+        python -m scripts.run_backend_tests""")
+
 _PARSER.add_argument(
```

Comment:
Use a [mutually exclusive group](https://docs.python.org/2/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group) to force test_path/test_target to be mutually exclusive.

Reply:
Done